ci/eval/diff.nix

```
@@ -76,11 +76,36 @@ let
   afterAttrs = getAttrs after;
   diffAttrs = diff beforeAttrs afterAttrs;
   diffJson = writeText "diff.json" (builtins.toJSON diffAttrs);
+
+  # The maintainer list is not diffed, but just taken as is, to provide a map
+  # of maintainers on the target branch. A list of GitHub IDs is sufficient for
+  # all our purposes and reduces size massively.
+  meta = lib.importJSON "${after}/${evalSystem}/meta.json";
+  maintainers = lib.pipe meta [
+    (lib.mapAttrsToList (
+      k: v: {
+        # splits off the platform suffix
+        package = lib.pipe k [
+          (lib.splitString ".")
+          lib.init
+          (lib.concatStringsSep ".")
+        ];
+        maintainers = map (m: m.githubId) v.maintainers or [ ];
```

I think most of us agree `or` is odd enough that we should add clarifying parens:

```suggestion
        maintainers = map (m: m.githubId) (v.maintainers or [ ]);
```

---

We could also `builtins.catAttrs "githubId"` if we don't care about throwing an eval error when a maintainer doesn't have a `githubId` attr. That should be fine since we enforce that elsewhere via the module system.
```suggestion
        maintainers = lib.catAttrs "githubId" (v.maintainers or [ ]);
```